content/language-agnostic/encyclopedia/what-is-a-parameter-and-an-argument?.md

```diff
@@ -0,0 +1,28 @@
+<!--
+---
+Subjects:
+  - "What is a parameter and an argument?"
+Tags: 
+  - "language-agnostic"
```

Comment:
I don't see this at author.codecademy.com/browse/tags
Totally open to creating an expanded list of tags for us to use, though--just post about it in the channel if you feel the need